userspace/libscap/scap.h

```diff
@@ -528,6 +530,9 @@ struct ppm_syscall_desc {
 #define UDIG_RING_SM_FNAME "udig_buf"
 #define UDIG_RING_DESCS_SM_FNAME "udig_descs"
 #define UDIG_RING_SIZE (8 * 1024 * 1024)
+#define UDIG_RING_CTRL_SOCKET_PATH "/var/run/udig.sock"
+#define UDIG_RING_CTRL_CONNECT_MAX_ATTEMPTS 500
```

Comment:
I don't believe this should be hardcoded, especially at a high-but-not-infinite value.

Reply:
Do you have any suggestion?